### PR TITLE
DA-3048: Add emoji lib

### PR DESCRIPTION
**Pull request by linonymous**
add emoji lib

### CODE REVIEW

File: emoji/emoji.go [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package emoji
+
+func getEmojiUnicode(emojiStr string) string {
```
Comment (ajinkyan83) [inline emoji/emoji.go:3]: Do we need to export this?

Comment (linonymous) [reply emoji/emoji.go]: yes. we need to export this function. Updated.

**Review by ajinkyan83 — APPROVED** [review]
/lgtm

File: emoji/emoji_test.go [hunk 2]
```diff
@@ -0,0 +1,32 @@
+package emoji
+
+import "testing"
+
+func Test_getEmojiUnicode(t *testing.T) {
```
Comment (enkhalifapro) [inline emoji/emoji_test.go:5]: No need to _ just name it `TestGetEmojiUnicode `

Comment (linonymous) [reply emoji/emoji_test.go]: Done!

**Review by enkhalifapro — APPROVED** [review]

**Pull request merged**